productivity-suite/piercing-library/src/piercing-fragment-host/piercing-fragment-host.ts

```diff
@@ -102,3 +105,13 @@ export class PiercingFragmentHost extends HTMLElement {
     });
   }
 }
+
+function updateFragmentsToPierce(operation: "++" | "--") {
```

Comment:
@petebacondarwin, what do you think about the operation argument there?

I sort of wonder if it is neat or awkward 😅